packages/ui-extensions/src/surfaces/customer-account/targets.ts

```diff
@@ -257,13 +257,6 @@ export type AllowedComponentsForRenderExtension<
 > = ExtractedAllowedComponentsFromRenderExtension<RenderExtensions[Target]>;
 
 export interface FullPageApi {
-  /**
-   * @deprecated This is deprecated. DO NOT USE. This will be removed soon without warning. use `api.navigation.currentEntry` and `api.navigation.addEventListener` instead.
-   */
-  location: StatefulRemoteSubscribable<{
-    pathname: string;
-    search: string;
-  }>;
```

Comment:
also here?  https://github.com/Shopify/ui-extensions/blob/6376b6f175e889bcca59c53ce17ab31321a185d1/packages/ui-extensions/src/surfaces/customer-account/api/docs.ts#L80

Reply:
Actually no wouldnt need to do that since we omit location